themes/codefor-theme/assets/scss/main.scss

```diff
@@ -259,3 +259,7 @@ blockquote {
     margin-bottom: 15px;
 }
 
+// hide element, e.g. twitter timeline
+.hidden {
+  display:none;
+}
```

Comment:
Please remove this, bootstrap already contains a helper class to set the `display` style property of elements to `none`. (`d-none`)